backend/config/settings/base.py

```diff
@@ -202,3 +195,12 @@
     "VIPPS_CALLBACK_PREFIX", default="https://xoff0kv3i3.execute-api.eu-north-1.amazonaws.com/default/Vipps_callback"
 )
 VIPPS_FALLBACK_PREFIX = env("VIPPS_FALLBACK_PREFIX", default="http://127.0.0.1:3000/ecommerce/fallback")
+
+
+# Celery
+CELERY_BROKER = env("CELERY_BROKER")
+CELERY_BACKEND = env("CELERY_BACKEND")
+CELERY_TASK_TRACK_STARTED = True
+CELERY_TASK_TIME_LIMIT = 30 * 60
+CELERY_TASK_RESULT_EXPIRES = env("CELERY_TASK_RESULT_EXPIRES")
+CELERY_BEAT_SCHEDULER = "django_celery_beat.schedulers.DatabaseScheduler"
```

Comment:
Should these include a `default=`? Also, should the template env file be extended?